plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -159,10 +126,53 @@ const Tabs = ( props: TabsProps ): JSX.Element => {
 	const marketplaceContextValue = useContext( MarketplaceContext );
 	const { selectedTab, setSelectedTab, hasBusinessServices } =
 		marketplaceContextValue;
-	const [ visibleTabs, setVisibleTabs ] = useState( getVisibleTabs( '' ) );
+	const { searchResultsCount } = marketplaceContextValue;
 
 	const query: Record< string, string > = useQuery();
 
+	const tabs: Tabs = {
+		search: {
+			name: 'search',
+			title: __( 'Search results', 'woocommerce' ),
+			showUpdateCount: false,
+			updateCount: 0,
+		},
+		discover: {
+			name: 'discover',
+			title: __( 'Discover', 'woocommerce' ),
+			showUpdateCount: false,
+			updateCount: 0,
+		},
+		extensions: {
+			name: 'extensions',
+			title: __( 'Extensions', 'woocommerce' ),
+			showUpdateCount: !! query.term,
```

Comment:
Personal preference, no need to change but just mentioning
```suggestion
			showUpdateCount: Boolean( query.term ),
```

Reply:
Interesting! Beyond preference, ChatGPT:

> If you prefer readability and clarity, Boolean(query.term) is better.
> If you prefer conciseness and slightly faster performance, !!query.term works well.

Always hard to choose between readability and performance :) But personally I don't find it `!!` _that_ unreadable

Reply:
Yup, `!!` is very common too which makes it just as readable